java/src/test/java/com/google/appengine/tools/pipeline/AsyncGCDExample.java

```diff
@@ -92,6 +96,9 @@ public void run() {
    */
   @SuppressWarnings("serial")
   public static class AskUserForNameJob extends Job0<String> {
```

Comment:
`@AllArgsConstructor`

No big deal, is just the example, but I guess it will fail